### PR TITLE
chore: add config for semantic release with GitHub Actions

### LINKED ISSUE #46: Move to GH Actions and disable Travis

### PR DESCRIPTION
**Pull request by alexandercbooth**
Closes #46. Working towards nteract/nteract#5500.

I think we'll still need to disable Travis for this repo and add the npm_token.

### CODE REVIEW

File: .github/workflows/publish-docs.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+name: Build and Deploy Docs
+
+on:
+  push:
+    branches: master
+
+jobs:
+  build-and-deploy:
```
Comment (captainsafia) [inline .github/workflows/publish-docs.yml:8]: Nice!

**Review by captainsafia — APPROVED** [review]
I've gone ahead and added the NPM_TOKEN secret to the repo. Let's give this a whirl!

**Comment by captainsafia** [comment]
And spoke too soon! We'll need to have an ipykernel installation available on the CI machine for the tests to run.

**Comment by captainsafia** [comment]
Alright -- this should be good now.

**Comment by alexandercbooth** [comment]
Whoops, thanks for taking care of that!

**Pull request merged**